stixcore/products/level0/science.py

```diff
@@ -119,14 +119,20 @@ def from_levelb(cls, levelb):
         control['index'] = 0
 
         data = Data()
-        data['start_time'] = (np.array(packets.get_value('NIX00404'), np.uint16)) * 0.1 * u.s
-        data['rcr'] = np.array(packets.get_value('NIX00401', attr='value'), np.ubyte)
-        data['integration_time'] = (np.array(packets.get_value('NIX00405'), np.int16)) * 0.1 * u.s
-        data['pixel_masks'] = _get_pixel_mask(packets, 'NIXD0407')
-        data['detector_masks'] = _get_detector_mask(packets)
+        data['start_time'] = packets.get_value('NIX00404').astype(np.uint16)
```

Comment:
I think this will introduce rounding issues now the cast was done before the multiplication by  * 0.1 but not it after
```suggestion
        data['start_time'] = packets.get_value('NIX00404')
```